index.html

```diff
@@ -14,10 +14,11 @@
     <h1> Number<span class='guesser-title'>Guesser</span></h1>
   </header>
   <section class="inputArea">
-    <input class="userInput" placeholder="Enter Your Guess" type="number"></input>
+    <input class="userInput" placeholder="Enter Your Guess" type="number">
```

Comment:
The `input` tag is a self-closing tag, so you don't need the `</input>` piece at the end.